core/SymbolRef.h

```diff
@@ -75,6 +75,29 @@ class SymbolRef final {
         return _id;
     }
 
+    inline bool isClassOrModule() const {
+        return kind() == Kind::ClassOrModule;
+    }
+
+    inline bool isMethod() const {
+        return kind() == Kind::Method;
+    }
+
+    inline bool isFieldOrStaticField() const {
+        return kind() == Kind::Field;
+    }
```

Comment:
Why is `Kind::Field` not `isField()`? Should we rename `Kind::Field` to `Kind::FieldOrStaticField`?

Why is the distinction between field and static field not reflected in the enum class?

Reply:
Good point. I can make that change. While working on this PR I regretted lumping fields and static fields together in my original break-up-symbol-table PR.

Reply:
Fixed.